core/frontend/src/quantity-formatting/QuantityFormatter.ts

```diff
@@ -237,21 +239,60 @@ export interface FormatterParserSpecsProvider {
 }
 
 /** Arguments sent to FormattingUnitSystemChanged event listeners.
- * @alpha
+ * @beta
  */
 export interface FormattingUnitSystemChangedArgs {
   // string that defines unit system activated.
   readonly system: UnitSystemKey;
 }
 
 /** Arguments sent to QuantityFormatsChanged event listeners.
- * @alpha
+ * @beta
  */
 export interface QuantityFormatsChangedArgs {
   // string that represents the QuantityType that has been overriden or the overrides cleared.
   readonly quantityType: string;
 }
 
+/** Arguments sent to UnitFormattingSettingsProvider when overrides are changed.
+ * @beta
+ */
+export interface QuantityFormatOverridesChangedArgs {
+  // string that represents the QuantityType that has been overriden or the overrides cleared.
```

Comment:
Use `/** jsdoc comments */` or the documentation tools will ignore them. Same problem elsewhere, e.g. UnitFormattingSettingsProvider.